packages/aws-cdk/test/integ/cli/test-vpc-lookup.sh

```diff
@@ -12,10 +12,20 @@ echo "Setup complete!"
 
 # verify we can synth the importing stack now
 echo "Verifying we can now import that VPC"
-ENABLE_VPC_TESTING="IMPORT" cdk synth -v ${STACK_NAME_PREFIX}-import-vpc
+FAILED=false
+if ! ENABLE_VPC_TESTING="IMPORT" cdk synth -v ${STACK_NAME_PREFIX}-import-vpc; then
+  # Don't stop here (still clean up) but remember that we failed
+  echo "[MARK] Importing failed! The error will be just above this line." >&2
+  FAILED=true
+fi
```

Comment:
Might be cleaner if we call `cdk destroy -f` at the beginning of the test. All of this additional conditional logic shouldn't be needed.

Reply:
Fair.